ntbs-service/Models/Entities/MDRDetails.cs

```diff
@@ -20,21 +20,22 @@ public partial class MDRDetails : ModelBase, IOwnedEntityForAuditing
         [Display(Name = "Relationship of the current case to the contact")]
         public string RelationshipToCase { get; set; }
         
-        [RequiredIf(@"CountryId == 235", ErrorMessage = ValidationMessages.NotifiedToPheStatusIsRequired)]
+        public bool IsCountryUK => Country?.IsoCode == Countries.UkCode;
+        [RequiredIf(@"IsCountryUK", ErrorMessage = ValidationMessages.NotifiedToPheStatusIsRequired)]
```

Comment:
when you manage to make it this concise, i like to use `RequiredIf(nameof(IsCountryUK), ...)` to make the annotation even more connected to the code :)